js/src/controls/SearchControl.js

```diff
@@ -52,10 +52,15 @@ export class LeafletSearchControlView extends control.LeafletControlView {
   leaflet_events() {
     if (this.model.get('layer') !== null) {
         var found_style = this.model.get('found_style');
+        var self = this;
         this.obj.on('search:locationfound', function(e) {
```

Comment:
You can use an arrow function here `(e) => { ... }`, this will allow you removing `var self = this;` and you can use `this` inside the function.

Reply:
Thanks, I did the change.